pkg/controller/signer/signer.go

```diff
@@ -0,0 +1,84 @@
+package signer
+
+import (
+	c "crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+	"log"
+)
+
+func newCertificateTemplate(certReq *x509.CertificateRequest) *x509.Certificate {
+
+	template := &x509.Certificate{
+		Subject: certReq.Subject,
+
+		SignatureAlgorithm: x509.SHA256WithRSA,
+
+		NotBefore:    time.Now().Add(-1 * time.Second),
+		NotAfter:     time.Now().Add(24 * time.Hour),
```

Comment:
Can we recover from an expired certificate? It's not uncommon for whole clusters to be shut down for days, especially in internal training environments. Or when :hankey: really happens.

Reply:
Apologies, I meant to set this to longer. Thanks for catching it.